.travis.yml

```diff
@@ -6,12 +6,15 @@ matrix:
     - os: linux
       compiler: clang
       env: CFLAGS="-std=c99 -O2 -Wall -Wextra -Werror -Weverything -Wno-#warnings -Wno-pedantic -Wno-padded -Wno-format-nonliteral -Wno-disabled-macro-expansion -Wno-missing-noreturn -Wno-reserved-id-macro -Wno-cast-align"
+           TEST_DNSTHREAD_ENABLE_REMOTE=1
     - os: linux
       compiler: gcc
       env: CFLAGS="-std=c99 -O2 -Wall -Wextra -Werror -Wpedantic -pedantic-errors -Wno-clobbered"
+           TEST_DNSTHREAD_ENABLE_REMOTE=1
     - os: osx
       compiler: clang
       env: CFLAGS="-std=c99 -O2 -Wall -Wextra -Werror -Weverything -Wno-#warnings -Wno-pedantic -Wno-padded -Wno-format-nonliteral -Wno-disabled-macro-expansion -Wno-missing-noreturn -Wno-reserved-id-macro"
+           TEST_DNSTHREAD_ENABLE_REMOTE=1
```

Comment:
Can't this be a global environment variable rather than being listed three times?

Reply:
done